### ISSUE TITLE
Is the tutorial out-dated?

### ISSUE THREAD

**Issue by demon450** [facebookarchive/caffe2#842]
For example, the minist tutorial:

    # do a simple stochastic gradient descent
    ITER = model.Iter("iter")
    # set the learning rate schedule
    LR = model.LearningRate(
        ITER, "LR", base_lr=-0.1, policy="step", stepsize=1, gamma=0.999 )

It looks like the model is a CNNModelHelper which is deprecated and if we run the current code it says no Iter found.
I made it work by using brew.Iter.

Also a lot of API does not have description. Is there a plan to fix this?
Thanks

**Comment by demon450** [comment]
Ok, I found the newest version in ipython. But it could be misleading for newcomers reading the webpage.
@aaronmarkham
